src/librustc_target/spec/powerpc_wrs_vxworks_spe.rs

```diff
@@ -20,7 +20,6 @@ pub fn target() -> TargetResult {
         options: TargetOptions {
```

Comment:
Move the following common settings from the target specific files to vxworks_base.rs:

- target_os
- target_vendor
- linker_flavor

Reply:
Done. I also moved target_env to vxworks_base as they are all "gnu".

Reply:
Actually I need to revert the change here. The vxworks_base.rs suppose to hold all "options", but 
they doesn't belong to "options".

Reply:
> Actually I need to revert the change here
OK that's unfortunate.